nsv13/code/modules/overmap/effects.dm

```diff
@@ -0,0 +1,29 @@
+//All credit goes to Goonstation's 2020 release for these explosion sprites, thanks goons!//
+
+/obj/effect/temp_visual/impact_effect/torpedo
+	icon = 'nsv13/goonstation/icons/effects/explosions/60x60.dmi'
+	icon_state = "explosion"
+	duration = 2 SECONDS
+
+/obj/effect/temp_visual/impact_effect/torpedo/Initialize()
+	icon_state = "[prob(50) ? "explosion" : "explosion2"]"
```

Comment:
This should probably be pick()d from a static list.

Reply:
Agree. Wouldn't prob(50) almost always be true since the default behavior is anything that's not null or 0 is true?